src/components/UI/StyledDropZone/StyledDropZone.tsx

```diff
@@ -75,24 +83,34 @@ export const StyledDropZone: FunctionComponent<DropZoneProps> = ({
 
       {error && <p className="max-w-lg text-rose text-lg leading-none font-bold mb-2">Error</p>}
       {fileTypeError && (
-        <p className="max-w-lg text-rose text-lg leading-none mb-2" data-testid="invalid-file-error">
-          Incorrect file type selected, Only {dropzoneOptions.accept} are allowed
-        </p>
+        <ErrorMessage
+          id="invalid-file-error"
+          message={`Incorrect file type selected, Only ${dropzoneOptions.accept} are allowed`}
```

Comment:
we could start consolidating these dropzone error codes + custom error messages somewhere, wdyt?

Reply:
yeah we can, most of them regarding the validation of documents are in the tt-utils repo. we can create another story to consolidate them.